Nos/Models/RelaySubscription.swift

```diff
@@ -28,9 +28,11 @@ struct RelaySubscription: Identifiable, Hashable {
         subscriptionStartDate != nil
     }
     
-    /// Returns true if this is a "one-time" filter, where we are only looking for a single event
-    var isOneTime: Bool {
-        filter.limit == 1
+    /// Whether this RelaySubscription should subscribe to all new events
+    /// matching the filter (if true) or close once a relay has returned all
+    /// stored events (if false).
+    var observeNewEvents: Bool {
```

Comment:
Maybe this is where my naming suggestion below should actually be used. Perhaps we could call this `keepAlive` or `keepOpen` or `persistentConnection` or something?

Reply:
keepAlive or keepOpen should still cause the code to contain the `!`, which you complained to in the last comment and I agree with. So we have to think about a better name that communicates the opposite. What do you think about `closesAfterResponse`.? @joshuatbrown

Reply:
Yeah, I think that could work.